src/ethereum/EthereumEngine.ts

```diff
@@ -991,10 +997,16 @@ export class EthereumEngine extends CurrencyEngine<
     }
 
     const edgeToken = tokenId != null ? this.builtinTokens[tokenId] : null
+    const networkBaseFeeWeiHex = await this.ethNetwork.getBaseFeePerGas()
+    const currentBaseFeeWei =
+      networkBaseFeeWeiHex != null
+        ? hexToDecimal(networkBaseFeeWeiHex)
+        : undefined
```

Comment:
Why don't you need to do hexToDecimal conversion in the other instance?

Reply:
Missed it, fixed